Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java

```diff
@@ -59,6 +60,19 @@ public CommandResult(String content, List<MessageEmbed> embeds, boolean send) {
             this.embeds = embeds;
             this.send = send;
         }
+        public CommandResult(String content, List<MessageEmbed> embeds, String name, boolean send) {
+            this.content = content;
+            this.embeds = embeds;
+            this.name = name;
+            this.send = send;
+        }
+        public CommandResult(String content, List<MessageEmbed> embeds, String name, String avatarUrl, boolean send) {
```

Comment:
send should be the third argument tbh, you will more often want to specifiy send than a custom profile

Reply:
Yeah but we have another overload for the constructor which doesn't accept any customizations for clyde https://github.com/Aliucord/Aliucord/blob/39ebdb38d34aeb837a55b299d2393b97cc6a0361/Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java#L58-L62  which is exactly for the case you're talking about

Reply:
but then it doesnt make sense that argument position changes in the overload, so i still think send should be the third param even in the overload with name and avatarUrl, and also maybe change name -> username

Reply:
Aight, gotcha.